helm/reana/templates/reana-workflow-controller.yaml

```diff
@@ -113,6 +113,10 @@ spec:
           {{- end }}
           - name: REANA_KUBERNETES_JOBS_MEMORY_LIMIT
             value: {{ .Values.kubernetes_jobs_memory_limit | default "4Gi" }}
+          - name: REANA_KUBERNETES_JOBS_MAX_USER_TIMEOUT_LIMIT
+            value: !!str {{ .Values.kubernetes_jobs_max_user_timeout_limit | default 1209600 }}  # 1209600 seconds = 14 days
```

Comment:
Ahá! here's the value. I still think it would be helpful to have it in the README.

Didn't know `!!str` 👌 but env vars are always transformed to string, aren't they?

Reply:
Agree. I only added it in the YAML file but it is hidden there. I will also add it to README.

Reply:
> Didn't know !!str 👌 but env vars are always transformed to string, aren't they?

Yes, env variables are strings. But, I think, this is a [matter of Helm templating](https://github.com/helm/helm/issues/4775#issuecomment-430361046). You can input numbers in two ways: `10` and `"10"`. The first one is treated as a number by templating engine (and, actually, needs the` quote` prefix before). The second one treats as a string, but you cannot use `quote` on that. I found `!!str` to convert whatever is input to string.

Reply:
I was thinking to allow for strings too. So, in the future, if we will go with "10h" suffixes it will require fewer changes. But, honestly, it was not that important for this PR set but I did what I did :smiley:

Reply:
It's OK for me, we can keep it.